h2o-algos/src/main/java/hex/generic/GenericModelParameters.java

```diff
@@ -22,6 +30,55 @@
      */
     public boolean _disable_algo_check;
 
+    /**
+     * Generic model parameters - might contain any parameters based on the state of the model in the training phase.
+     */
+    public ModelParameterSchemaV3[] _modelParameters;
+
+    protected static ModelParameterSchemaV3[] convertParameters(final ModelParameter[] originalParams) {
+        final ModelParameterSchemaV3[] convertedParams = new ModelParameterSchemaV3[originalParams.length];
+
+        for (int i = 0; i < originalParams.length; i++) {
+            final ModelParameter originalParam = originalParams[i];
+            final ModelParameterSchemaV3 convertedParam = new ModelParameterSchemaV3();
+            // Hand-built mapping for better performance compared to reflection
+            convertedParam.name = originalParam.name;
```

Comment:
some of these parameters are only good for model building (eg. gridable) - should we store these as well?

Reply:
It gives our users the full view. This is the only argument I can find. Besides unknown use-cases from distant future, like upgrading Mojo to a full-fledged generic model container, capable of checkpointing. Then, these could be useful. Not sure if that would be backwards compatible all the way.

Reply:
Not exposing old model_id. As that is irrelevant and hard to expose in a non-confusing way. The imported generic model has it's own model_id. It can be added any time later.